modules/microshift-olm-deploy-ops-con.adoc

```diff
@@ -0,0 +1,32 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-deploy-operators_{context}"]
+= How to deploy Operators
+
+After you create and deploy your custom catalog, you must create a Subscription custom resource (CR) that can access the catalog and install the Operators you choose. Where Operators run depends on both the namespace the catalog runs in and the namespace in which you create the Subscription CR.
+
+[id="microshift-operators-namespace-details_{context}"]
+== Namespace details
+The `microshift-olm` RPM creates the three default namespaces:
+
+* OLM runs by default in the `openshift-operator-lifecycle-manager` namespace.
+* Operators run by default in the `openshift-operators` namespace.
+* The global namespace is `openshift-marketplace`. This namespace is empty by default and must be set in the catalog source YAML.
+* Operators run in the same namespace as the catalog unless the catalog is deployed to the global `openshift-marketplace` namespace. Catalogs added to the global namespace can be referenced from any other namespace.
+* If you want to use an Operator installed in a different namespace, or in more than one namespace, both the catalog source and the Subscription CR that references the Operator must be running in the `openshift-marketplace` namespace.
```

Comment:
I think that namespace of the `Subscription` defines where the Operator runs.

With CatalogSource in openshift-marketplace, if I create Sub in openshift-marketplace, Operator will run in openshift-marketplace; if I create Sub in openshift-operators, Operator will run in openshift-operators